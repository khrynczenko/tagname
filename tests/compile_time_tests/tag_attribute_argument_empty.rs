use tagname::TagName;

#[derive(TagName)]
enum Cases {
    #[tag]
    Lower,
    Upper
}

fn main() {
}
