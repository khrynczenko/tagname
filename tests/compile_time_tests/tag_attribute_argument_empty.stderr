error: `tag` attribute expects an assignment expression `[tag(case = "lower" | "upper")]`
 --> tests/compile_time_tests/tag_attribute_argument_empty.rs:5:5
  |
5 |     #[tag]
  |     ^
